app/assets/stylesheets/application.scss

```diff
@@ -5,18 +5,12 @@
 // https://design.education.gov.uk/design-system/dfe-frontend
 // https://design-system.service.gov.uk/styles/colour/
 
-// https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss
-
 
 $govuk-new-link-styles: true;
 $govuk-page-width: 1200px;
```

Comment:
Is this needed?

Reply:
I have put this back in as it does have some useful information